docs/docs/Developing_with_JAC/Language_Feature.md

```diff
@@ -20,7 +20,7 @@ std.input();
 ```
 
 ## Data types
-Jaseci is a dynamically typed language so there is no need to specify the data type.
+Jaseci is a dynamically typed language so there is no need to declare the data type.
```

Comment:
JAC is a dynamically typed language so there is no need to declare the data type.